src/frontend/bibletime_init.cpp

```diff
@@ -546,12 +546,12 @@ void BibleTime::initActions() {
             &m_actionCollection->action(QStringLiteral("autoTabbed"));
     m_windowAutoTabbedAction->setCheckable(true);
 
-    //: Vertical tiling means that windows are vertical, placed side by side
+    // Vertical tiling means that windows are vertical, placed side by side
     m_windowAutoTileVerticalAction =
             &m_actionCollection->action(QStringLiteral("autoVertical"));
     m_windowAutoTileVerticalAction->setCheckable(true);
 
-    //: Horizontal tiling means that windows are horizontal, placed on top of each other
+    // Horizontal tiling means that windows are horizontal, placed on top of each other
```

Comment:
The `//:` comment lines were already removed by commit 6b2ee7454fa885d30d10366dc57e9ca792a77eee so this commit is longer needed.`